tests/network/curl_server.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2015-2018 SUSE Linux GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Server configurations specifit to test curl .
+# Maintainer: Marcelo Martins <mmartins@@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use mm_network 'setup_static_mm_network';
```

Comment:
not needed

Reply:
I guess in all your files as it is done in `setup_multimachine.pm`

Reply:
ok, done! Thanks